js/summary_test.go

```diff
@@ -55,7 +55,10 @@ func TestTextSummary(t *testing.T) {
 			runner, err := getSimpleRunner(
 				t, "/script.js",
 				fmt.Sprintf(`
-					exports.options = {summaryTrendStats: %s};
+					exports.options = {
+						summaryTrendStats: %s,
+						handleSummaryTimeout: '120s',
+					};
```

Comment:
Do you really need to update all of those tests ?

Reply:
I think the defaults are set at the `cmd` level. But, these unit tests are testing the functionality of these respective modules independent of `cmd` execution.

Reply:
@mstoykov : Friendly ping on this! 😄